app/components/participant-creator/template.hbs

```diff
@@ -45,7 +45,7 @@
                                     <label class="text-muted">
                                         Use as password?
                                     </label>
-                                    <input name="useAsPassword" onchange={{action (action 'useAsPassword' item.key) value="target.checked"}} type="radio" />
+                                    <input name="useAsPassword" checked={{if useAsPassword true}} onchange={{action (action 'useAsPassword' item.key) value="target.checked"}} type="radio" />
```

Comment:
I tried to create one account, tag `ATAG`, Additional fields `A-field`

I then told it to use that extra field as password, and clicked "create accounts".

I got the following error on the console, then the page hung- it does not seem to deal with failures on the server.

```
Saving nOte4-ATAG
bcrypt.js:172 Uncaught Error: Illegal arguments: object, string
```